### PR TITLE
Clarify loss epoch

### LINKED ISSUE #3210: 3.1.3 loss epoch
> QUIC ends a loss epoch when a packet sent after loss is declared is acknowledged. 

I believe there's an extra "is".

### PR DESCRIPTION
**Pull request by ianswett**
Fixes #3210

### CODE REVIEW

File: draft-ietf-quic-recovery.md [hunk 1]
```diff
@@ -217,7 +217,7 @@ not available.
 
 ### Clearer Loss Epoch
 
-QUIC ends a loss epoch when a packet sent after loss is declared is
+A loss epoch ends when a packet sent after loss is first declared is
```
Comment (marten-seemann) [inline draft-ietf-quic-recovery.md]: I don't understand this sentence. There are too many "is".

Comment (ianswett) [reply draft-ietf-quic-recovery.md]: Fair, I tried to rephrase this sentence another time.  PTAL

File: draft-ietf-quic-recovery.md [hunk 2]
```diff
@@ -217,12 +217,13 @@ not available.
 
 ### Clearer Loss Epoch
 
-QUIC ends a loss epoch when a packet sent after loss is declared is
-acknowledged. TCP waits for the gap in the sequence number space to be filled,
-and so if a segment is lost multiple times in a row, the loss epoch may not
-end for several round trips. Because both should reduce their congestion windows
-only once per epoch, QUIC will do it correctly once for every round trip that
-experiences loss, while TCP may only do it once across multiple round trips.
+QUIC starts a loss epoch when a packet is lost and ends one when any packet
```
Comment (marten-seemann) [inline draft-ietf-quic-recovery.md]: s/one/it

Comment (ianswett) [reply draft-ietf-quic-recovery.md]: I actually think one reads more clearly, but maybe that's me?

**Comment by rpaulo** [comment]
I was trying to come up with text for this but this PR looks great.

**Pull request merged**